datajoint/__init__.py

```diff
@@ -71,17 +73,19 @@ class key:
 from .errors import DataJointError, DuplicateError
 
 
-def create_virtual_module(modulename, dbname):
+def create_virtual_module(module_name, schema_name, create_schema=False, create_tables=False):
```

Comment:
What would be the reason to have `create_tables` default to `False`? I totally agree with making `create_schema=False` default but don't see strong reason to prevent usage of schema in defining additional tables in the schema?

Reply:
I don’t have an opinion either way.  This use case is rare and I think should be discouraged in favor of the more common approach using the dj.schema.   Users can explicitly set create_tables=True if they do intend to add new tables.

Reply:
I guess the idea is to treat the `dj.create_virtual_module` to be by default a "read-only" schema module? I guess that could make sense.

Reply:
Alright have a couple more minutes of thought I don't have a strong reason against this default "read-only" behavior for virtual modules, especially given that user does have an option to set create tables to true.

Reply:
Let's merge then and make a release.